scripts/linters/pylint_extensions_test.py

```diff
@@ -3272,3 +3272,96 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class StringConcatenationCheckerTests(unittest.TestCase):
```

Comment:
One more small suggestion - Please include one test to check whether the error msg string is being correctly generated or not - i.e the generated error string is exactly equal to what we expect or not. (We are slowly trying to enforce this practice in our codebase)

Reply:
> Hey @tanishq67! Is there any update on this PR? Let me know if you are stuck anywhere. Also, left one small comment for you.

I was quite sick, I will update it in a day or so.

Reply:
>One more small suggestion - Please include one test to check whether the error msg string is being correctly generated or not - i.e the generated error string is exactly equal to what we expect or not. (We are slowly trying to enforce this practice in our codebase)

@sajalasati as per your previous suggestion, I will be converting the message to a static message. Don't you think we should keep this test for only dynamic messages?

Reply:
@tanishq67 is this comment resolved?

Reply:
>@tanishq67 is this comment resolved?

Please check this [comment](https://github.com/oppia/oppia/pull/12296#issuecomment-825157024)

Reply:
Thanks for the link, please make sure to reply in the thread so that the thread is updated!